src/DocumentDB.ChangeFeedProcessor.IntegrationTests/EstimatorTests.cs

```diff
@@ -25,8 +25,6 @@ public EstimatorTests(IntegrationTestFixture fixture) : base(fixture, typeof(Est
         [Fact]
         public async Task CountPendingDocuments()
         {
-            // Cleanup the test collection to avoid other tests' documents causing issues with StartFromBeginning
-            await this.ResetTestCollection();
             int documentCount = 1;
             int partitionCount = await IntegrationTestsHelper.GetPartitionCount(this.ClassData.monitoredCollectionInfo);
```

Comment:
can we remove TestClassData entirely?

Reply:
`TestClassData` is used to share the collections' information across tests of the same file, since these are generated based on the test file type and names, they are shared across tests and reused through a Fixture.

Reply:
Removed TestClassData